### ISSUE TITLE
images not found when rendering PDF document using intermediates dir

### ISSUE THREAD

**Issue by aronatkins** [rstudio/rmarkdown#438]
This is a duplicate of #429 but specifically when using `intermediates_dir` to generate PDFs.

Using the same document:

```
![rstudio](blue-250.png)
```

but now with the header YAML:

```
---
output: pdf_document
---
```

I get the following error when using `intermediates_dir`:

```
pandoc: Could not find image `blue-250.png', skipping...
! Package pdftex.def Error: File `blue-250.png' not found.
```

Without `intermediates_dir`, rendering is successful.

**Comment by jjallaire** [comment]
Hopefully if we can do what I suggested re: parsing the markdown this particular error case will go away. @jmcphers Does that make sense?

**Comment by aronatkins** [comment]
Agreed (parsing the markdown to extract deps). Just wanted to make sure I documented the various layers of the problem.

**Comment by aronatkins** [comment]
Confirmed fixed with #442.
